MONEY/NetworkSpentUpgradeAgency.md

```diff
@@ -0,0 +1,20 @@
+---
+ns: MONEY 
+---
+
+## _NETWORK_SPENT_UPGRADE_AGENCY
+
+```c
+// 0x6CCA64840589A3B6 
+void _NETWORK_SPENT_UPGRADE_AGENCY(Any p0, Any p1, Any p2, Any p3);
+```
+
+```
+NativeDB Introduced: v2545
+```
+
+## Parameters
+* **p0**:
+* **p1**:
+* **p2**:
+* **p3**:
```

Comment:
Param's seems to be BOOL `amount`, BOOL `if from bank AND wallet`, CHAR `context`, and CHAR `option` that seems to be 0 by default.